### PR TITLE
CVE-2010-2646 and CVE-2013-2854

### PR DESCRIPTION
**Pull request by yojake**

### CODE REVIEW

**Comment by andymeneely** [comment]
Hi - I didn't see this username in our survey. If you're a 331 student, please make sure you go to [Step 2 of the assignment](https://github.com/VulnerabilityHistoryProject/chromium-vulnerabilities/tree/dev/assignment).

**Comment by yojake** [comment]
I've already submitted the survey and cannot again. The wording of the instructions made it sound like this was optional. My RIT username is jcy5214

**Comment by andymeneely** [comment]
Ah! Hi Jacob. Yes, I saw your survey response. It is optional. You can send your work via email to Prof. Murukanaiah and he'll work it out there. You are also welcome to delete this pull request. 

Sorry - that's my fault. I didn't pass the right instructions to Prof. Murukannaiah

File: cves/CVE-2010-2646.yml [hunk 1]
```diff
@@ -95,20 +101,20 @@ discovered:
...
   question: |
     What subsystems was the mistake in?
 
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged. Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer: 
-  name: 
+  answer: Based on the description in the CVE and bug forum posts
+  name: Blink, WebKit
```
Comment (Kosci) [inline cves/CVE-2010-2646.yml]: renderer as well?

File: cves/CVE-2010-2646.yml [hunk 2]
```diff
@@ -47,16 +48,21 @@ reviews:
...
+- :commit: 13067841a467f023546a9d7a8a4e74ff91a583cb
   :note: ''
 vccs:
-- :commit:
-  :note: 
+- :commit: daa8c58ee49795d292312bbf217999c93693919c
+  :note: This was new code to move responsibilities into a different component.
+- :commit: 7aac4496431a922790d2b7472c8309a238f145cf
+  :note: This was the inital commit for the StorageEventDispatcher, code remained mostly untouched.
 upvotes_instructions: |
```
Comment (Kosci) [inline cves/CVE-2010-2646.yml]: +5 upvotes, very interesting issue

File: cves/CVE-2013-2854.yml [hunk 3]
```diff
@@ -56,8 +56,8 @@ fixes:
 - :commit: 6f62e3b41fe9ef708aa8c990dd814ae91d68f2f4
   :note: ''
```
Comment (Kosci) [inline cves/CVE-2013-2854.yml]: in the commit note, mention that it's a check for invalid handle value

File: cves/CVE-2013-2854.yml [hunk 4]
```diff
@@ -56,8 +56,8 @@ fixes:
...
 vccs:
-- :commit:
-  :note: 
+- :commit: e7f009da9894df2b8e48aa21578aa4e5ecd7a765
+  :note: 'This was the creation of a new suite of classes for MHTML generation'
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
```
Comment (Kosci) [inline cves/CVE-2013-2854.yml]: +1 upvote

**Pull request closed without merging**